tests/testAutopas/tests/selectors/AutoTunerTest.cpp

```diff
@@ -25,17 +25,23 @@ TEST_F(AutoTunerTest, testAllConfigurations) {
                                                  autopas::SelectorStrategy::fastestAbs, 100, maxSamples);
 
   autopas::Logger::get()->set_level(autopas::Logger::LogLevel::off);
+  //  autopas::Logger::get()->set_level(autopas::Logger::LogLevel::debug);
   bool stillTuning = true;
   auto prevConfig = autopas::Configuration(autopas::ContainerOption(-1), autopas::TraversalOption(-1),
                                            autopas::DataLayoutOption(-1), autopas::Newton3Option(-1));
 
+  // number of possible configurations * number of samples + last iteration after tuning
+  size_t expectedNumberOfIterations = 34 * maxSamples + 1;
```

Comment:
document more clearly how the number 34 has been calculated, as it will be subject to change!